.travis.yml

```diff
@@ -20,7 +20,7 @@ script:
   - xcodebuild -showsdks
 
   # Build Framework in Release
-  - xcodebuild clean build -project "$XCODEPROJ" -scheme "$SCHEME" -destination "$DESTINATION" -configuration Release | xcpretty;
+  - xcodebuild clean build -project "$XCODEPROJ" -scheme "$SCHEME" -destination "$DESTINATION" -configuration Release ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test | xcpretty;
```

Comment:
It does, but it's always good to do both separately to make sure everything is ok in their right configuration